zvmsdk_sles.spec

```diff
@@ -68,8 +72,18 @@ chown zvmsdk /etc/zvmsdk/token.dat
 chgrp zvmsdk /etc/zvmsdk/token.dat
 chmod 0600 /etc/zvmsdk/token.dat
 
+if systemctl is-active --quiet firewalld; then
+    echo "Firewalld is running, opening port 8080 in the docker zone"
+    firewall-cmd --zone=docker --permanent --add-port=8080/tcp
+    firewall-cmd --zone=docker --add-port=8080/tcp
```

Comment:
I have doubts about `--zone=docker`. This might be specific to linux1 SLES images :grimacing: .

Here too, we need to test on the VMs provided by Mike before we know for sure our firewall code is correct.

Reply:
Did you experiment without the --zone=docker option on level 2 VMs ?

Reply:
Testing is done successfully